osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -145,33 +146,63 @@ public async Task TestImportThenExportThenImport()
                     var osu = LoadOsuIntoHost(host);
 
                     var temp = TestResources.GetTestBeatmapForImport();
-                    string extractedFolder = $"{temp}_extracted";
-                    Directory.CreateDirectory(extractedFolder);
 
                     //Full path to the export folder created in Temp\of-test-headless
-                    string exportFullPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Temp\\of-test-headless\\TestImportThenExportThenImportImportBeatmapTest\\exports");
+                    string exportFullPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Temp\\of-test-headless\\TestExportBeatMapWithInvalidCharactersPathImportBeatmapTest\\exports");
```

Comment:
this will be my last comment on this review but this is wrong. you can't just assume a temp path directory location like this. follow other examples, read existing code, write good code.